perception_pipeline_config:
  scene_bounds: 
    min: [-1.5, -1.5, -1.5]
    max: [1.5, 1.5, 1.5]
